packages/sdk/src/utils/blockExplorer.ts

```diff
@@ -6,9 +6,9 @@ export function getBlockExplorerLink({
   providerUrl,
 }: {
   path: string;
-  providerUrl: string;
+  providerUrl?: string;
 }) {
-  return `${BLOCK_EXPLORER_URL}${path}?providerUrl=${encodeURIComponent(
-    providerUrl
-  )}`;
+  return `${BLOCK_EXPLORER_URL}${path}${
+    providerUrl ? `?providerUrl=${encodeURIComponent(providerUrl)}` : ''
```

Comment:
I think we should not make `providerUrl` optional as we want to enforce we are passing it when opening block explorer.

Reply:
- this is a util function for the whole app, it should accept optional and send to default provider. it's up
- this function is just temporary, we're gonna use from SDK when [fuels-ts/309](https://github.com/FuelLabs/fuels-wallet/pull/309) gets merged


if you think `providerUrl` should be always informed, [this](https://github.com/FuelLabs/fuels-wallet/pull/309) is the right place to ask for it

Reply:
On SDK level it should not be required but here yes